src/main/scala/nl/knaw/dans/easy/stage/package.scala

```diff
@@ -27,6 +27,20 @@ package object stage {
 
   val xsds: Array[String] = Array(NameSpace.DC.uri, NameSpace.DDM.uri)
 
+  type ObjectType = String
+
+  type ObjectKey = String
+
+  type PathInDataset = String
+
+  /**
+   * ObjectKey is a SDO folder name if ObjectType == "objectSDO"
+   * ObjectKey is a URI (info:fedora/easy-f...:...) if ObjectType == "object"
+   */
+  type RelationObject = (ObjectType, ObjectKey)
```

Comment:
* you declare some rules (invariants) of this type alias in the comments. I'd rather see them encoded in the type itself (hint: for that you need to make it a case class, I suppose).
* Do I read it correctly that `ObjectType` can have two values only, namely `"objectSDO"` and `"object"`? That is not reflected by the type of `ObjectType = String` above! If this is the case, either make an Enum or (my preference) a sealed abstract class with two case objects.